modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/vulcan/openapi/contributor/ObjectEntryOpenAPIContributor.java

```diff
@@ -114,56 +114,61 @@ else if (key.contains("objectRelationshipName")) {
 				for (Map.Entry<ObjectRelationship, ObjectDefinition> entry :
 						relatedObjectDefinitionsMap.entrySet()) {
 
-					ObjectRelationship objectRelationship = entry.getKey();
-
 					ObjectDefinition relatedObjectDefinition = entry.getValue();
 
-					String relatedSchemaName = getSchemaName(
-						relatedObjectDefinition);
+					if (relatedObjectDefinition.isActive()) {
```

Comment:
Instead of wrapping all the code inside the if, do the inverse to save an indentation level:

```java
if (!relatedObjectDefinition.isActive()) {
   continue;
}
```

This way the code reads better.

Reply:
Make sense @4lejandrito!!